### ISSUE TITLE
Navigation to URLs with # should focus on that respective section of the webpage

### ISSUE THREAD

**Issue by jaredwyee** [cagov/design-system#661]
Presently, our "jump to" or "focus on this section" URL behavior does not focus on that section of a webpage. For example if you go to https://designsystem.webstandards.ca.gov/style/content/#our-inspiration the focus is NOT at the `Our inspiration` header. 

This presents an accessibility tabbing issue. If you were to tab to `Our inspiration` > hit Enter... at next tab the expectation would be to be focused on **18F**. However if you hit tab, you're focused on `Share your thoughts`

This behavior is present across all pages with a left nav. 

![image](https://user-images.githubusercontent.com/85529382/153978394-cac5e8ab-3100-4dad-8328-9da8b938f942.png)

**Issue:**
Tab focus shifts back to the local navigation instead of the link below the section selected.

**Recommendation**
Tab should begin from the section that has focus and not wrap to the top of the page once a local navigation link is clicked.

**WCAG 2.1 Guidelines**
[2.4.3 Focus Order (Level A)
](https://www.w3.org/TR/WCAG21/#focus-order)
[3.2.1 On Focus (Level A)](https://www.w3.org/TR/WCAG21/#on-focus)

**Comment by jaredwyee** [comment]
Confirmed working. Closing. Pending move to done once merge completed tomorrow.
